node/tessel-export.js

```diff
@@ -917,6 +917,8 @@ Tessel.SPI = function(params, port) {
 
   this.chipSelectActive = params.chipSelectActive === 'high' || params.chipSelectActive === 1 ? 1 : 0;
 
+  this.chipSelectDelay = params.chipSelectDelayUs/10 || 0;
```

Comment:
this.chipSelectdelay is passed to setTimeout, so it's in ms. This should be `/1000`?